### DISCUSSION TITLE
Reorder pagination controls on the table bottom

### DISCUSSION THREAD

**Discussion by norbertpapp** [quasarframework/quasar#12919] [General - Components / Directives / etc]
![image](https://user-images.githubusercontent.com/186974/159917059-405bc2d8-d9f0-4dd0-9844-19c832a0da73.png)
Is there an option to reorganize the elements on the bottom?
I have to move the number of rows _(and only that, not including the pagination)_ to the left side of the table footer, but I don't see any options in the documentation that would help me do that.
Do you have any ideas to solve this?

**Comment by MilosPaunovic** [comment]
You can use pagination slot.
[https://quasar.dev/vue-components/table#pagination-slot](https://quasar.dev/vue-components/table#pagination-slot)

**Reply by norbertpapp** [reply]
Thanks for the swift answer, I'll try it 🙏
